src/libs/OptionsListUtils.js

```diff
@@ -407,16 +407,21 @@ function getSidebarOptions(reports, personalDetails, draftComments, activeReport
  *
  * @param {Boolean} hasSelectableOptions
  * @param {Boolean} hasUserToInvite
+ * @param {String} searchValue
  * @param {Boolean} [maxParticipantsReached]
  * @return {String}
  */
-function getHeaderMessage(hasSelectableOptions, hasUserToInvite, maxParticipantsReached = false) {
+function getHeaderMessage(hasSelectableOptions, hasUserToInvite, searchValue, maxParticipantsReached = false) {
     if (maxParticipantsReached) {
         return CONST.MESSAGES.MAXIMUM_PARTICIPANTS_REACHED;
     }
 
     if (!hasSelectableOptions && !hasUserToInvite) {
-        return CONST.MESSAGES.NO_CONTACTS_FOUND;
+        if (/^\d+$/.test(searchValue)) {
```

Comment:
Does this check if it starts with a `0` ? I don't see it here.

Reply:
If it starts with a 0, the country code validation function runs, the function is already on source code.
Thanks for your kindness.

Reply:
Yes, you're right.
If it starts with a '0', the country code validation function runs. The function exist already in source code.
Thanks for your kindness.